### DISCUSSION TITLE
Supported DBs?

### DISCUSSION THREAD

**Discussion by dstango** [sproket/Persism#10] [Q&A] [answered]
Hi Dan,

I just stumbled upon your project and it seems very interesting, so I would like to try it out. Is it possible to use other DBs beside the ones defined in ConnectionTypes (there's an enum "Other" ....)? I will have to interface an Informix DB ... 
Should it just work "out of the box" (at least for simple cases), or do I need to configure something somewhere?

Thanks!

Dirk

**Comment by sproket** [comment, marked as answer]
Hi! Thanks for the interest!

It should work. The connection types define things like keyword name delimiters for column names like [] for SQL or "" for Oracle, etc.. 

The only issue you could have would be if you have column names with spaces or column names that are keywords.

I've been meaning to add Informix. Is there a developer edition for download?  I could include it in my upcoming release.

In any case, you could grab the 1.0.2 snapshot and add Informix to ConnectionTypes as a work around for the moment.

Hope this helps

**Reply by dstango** [reply]
Thanks for the prompt reply, that sounds good!

As spaces/keywords in column names are edge cases I try to avoid it shouldn't be a problem for me right now.

And yes, there's a developer edition available: https://www.ibm.com/products/informix/editions

I'll give it a try then :-)
